app/database/seeds/data/question.json

```diff
@@ -1,60 +1,60 @@
+{"id": 40, "question_html": "Are family or unisex bathrooms available?", "question_category_id": 5, "order": 2, "is_always_required": true, "explanation": null, "is_required_config": null, "name": "Family/Unisex Bathroom Availablity"},
+{"id": 41, "question_html": "Is the doorway wide enough to allow wheelchair access(width over 38 inches or 96 cm)?", "question_category_id": 5, "order": 3, "is_always_required": true, "explanation": null, "is_required_config": null, "name": "Wheelchair Access - Wide Doorway"},
+{"id": 42, "question_html": "Is the access button or automatic door working? (Answer Yes for doorless entrance)", "question_category_id": 5, "order": 4, "is_always_required": true, "explanation": null, "is_required_config": null, "name": "Automatic Door"},
+{"id": 43, "question_html": "Is the stall large enough to turn a wheelchair or walker?", "question_category_id": 5, "order": 6, "is_always_required": true, "explanation": null, "is_required_config": null, "name": "Wheelchair Access - Stall"},
+{"id": 44, "question_html": "Are grab bars mounted beside the toilet?", "question_category_id": 5, "order": 7, "is_always_required": true, "explanation": null, "is_required_config": null, "name": "Grab Bars"},
+{"id": 45, "question_html": "Is there leg clearance under the sink for wheelchair access?", "question_category_id": 5, "order": 9, "is_always_required": true, "explanation": null, "is_required_config": null, "name": "Wheelchair Access - Under The Sink"},
+{"id": 46, "question_html": "Is the counter or sink low enough for wheelchair access?", "question_category_id": 5, "order": 10, "is_always_required": true, "explanation": null, "is_required_config": null, "name": "Wheelchair Access - Counter"},
+{"id": 47, "question_html": "Do taps have lever handles or working motion sensors?", "question_category_id": 5, "order": 11, "is_always_required": true, "explanation": null, "is_required_config": null, "name": "Grab Bars - Motion Sensors"},
+{"id": 48, "question_html": "Is there a working sensor or one hand activated soap dispenser?", "question_category_id": 5, "order": 12, "is_always_required": true, "explanation": null, "is_required_config": null, "name": "Motion Sensors - Soap Dispenser"},
+{"id": 49, "question_html": "Is there a working sensor hand dryer (paper or hot air)?", "question_category_id": 5, "order": 13, "is_always_required": true, "explanation": null, "is_required_config": null, "name": "Motion Sensors - Hand Dryer"},
+{"id": 50, "question_html": "Is there a large weight capacity change table in a private area?", "question_category_id": 5, "order": 14, "is_always_required": false, "explanation": null, "is_required_config": "{\"is_required_if_yes_to\": 39}", "name": "Change Table"},
+{"id": 51, "question_html": "Is a ceiling track lift available?", "question_category_id": 5, "order": 15, "is_always_required": false, "explanation": null, "is_required_config": "{\"is_required_if_matching_tag\": [4]}", "name": "Ceiling Track Lift"},
```

Comment:
Fix some spelling errors with "Availablity".  It should be "Availability" instead.